crates/network-scanner/Cargo.toml

```
@@ -11,7 +11,7 @@ anyhow = "1"
 crossbeam = { version = "0.8", features = ["crossbeam-channel"] }
 dns-lookup = "2.0"
 mdns-sd = "0.10"
-network-interface = "1.1"
+network-interface = "2.0.0"
```

suggestion: Avoid specifying the patch number. We may over-constrain our dependency tree. 
```suggestion
network-interface = "2.0"
```